.github/workflows/commitMessage.yml

```
@@ -0,0 +1,31 @@
+name: Reusable Check Commit Message Workflow
+
+on:
+  workflow_call:
+    secrets:
+      CUSTOM_GITHUB_TOKEN:
+        required: true
+
+jobs:
+  check-commit-message:
+    name: Check Commit Message
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get PR Commits
+        id: 'get-pr-commits'
+        uses: tim-actions/get-pr-commits@master
+        with:
+          token: ${{ secrets.CUSTOM_GITHUB_TOKEN }}
+      - name: Check Subject Line Length
+        uses: tim-actions/commit-message-checker-with-regex@v0.3.1
+        with:
+          commits: ${{ steps.get-pr-commits.outputs.commits }}
+          pattern: '^.{0,50}(\n.*)*$'
+          error: 'Subject too long (max 50)'
+      - name: Check Body Line Length
+        if: ${{ success() || failure() }}
+        uses: tim-actions/commit-message-checker-with-regex@v0.3.1
+        with:
+          commits: ${{ steps.get-pr-commits.outputs.commits }}
+          pattern: '^.+(\n.{0,72})*$'
+          error: 'Body line too long (max 72)'
```

In the POSIX standard, a line is defined as: `A sequence of zero or more non- <newline> characters plus a terminating <newline> character`. We should follow this standard and add a newline at the end of a file.

```suggestion
          error: 'Body line too long (max 72)'
```